setup.py

```diff
@@ -12,13 +12,15 @@
 with open(os.path.join(this_dir, "README.md"), "r") as file:
     long_description = file.read()
 
+with open(os.path.join(this_dir, "version.txt"), "r") as file:
+    __version__ = re.findall('\d*\.\d*\.\d*', file.read())[0]
+    print("version: ",__version__)
```

Comment:
Not sure print statement is necessary.

Reply:
Indeed, was a debug print statement; will be removed.